src/jwplayer/utils/usePlaylistItem.ts

```diff
@@ -2,9 +2,9 @@ import { useContext, useState, useEffect } from 'react';
 import { PlaylistItem, PlaylistItemPlayerEventData } from 'jwplayer/types';
 import { PlayerContext } from 'jwplayer/players/shared/PlayerContext';
 
-export default function usePlaylistItem(): PlaylistItem {
+export default function usePlaylistItem() {
 	const { player } = useContext(PlayerContext);
-	const [playlistItem, setPlaylistItem] = useState<PlaylistItem>({ title: undefined, duration: undefined });
+	const [playlistItem, setPlaylistItem] = useState<PlaylistItem>();
```

Comment:
Make `playlistItem` undefined by default. VideoDetails will not render NaN:NaN then.